mmv1/third_party/terraform/tests/resource_monitoring_slo_test.go

```diff
@@ -410,6 +410,28 @@ func TestAccMonitoringSlo_windowBasedMetricSumRangeSlis(t *testing.T) {
 	})
 }
 
+func testAccMonitoringSlo_genericService(t *testing.T) {
```

Comment:
This doesn't seem to use the generic_service resource. It also isn't executed because it needs to start with `TestAcc` instead of `testAcc`

Can you add a new test file for the generic service resource, and add a test that updates the resource? It will need two or more steps with configs